app/e/[eventId]/components/LatestNewsSection.tsx

```
@@ -0,0 +1,59 @@
+import { fetchNews } from "@lib/api/news";
+import NewsCard from "@components/ui/newsCard";
+import Link from "next/link";
+import { formatCatalanA } from "@utils/helpers";
+
+interface LatestNewsSectionProps {
+  placeSlug: string;
+  placeLabel: string;
+  placeType: "region" | "town";
+  newsHref: string;
+}
```

ESLint

[error] Do not declare interfaces outside of the /types directory. Centralize all interfaces in /types.

(no-restricted-syntax)